Update outdated info about .NET Core WinForms designer (#16403)
dotnet/docs: docs/core/porting/winforms.md

## Prerequisites

> - [Visual Studio 2019 16.5 Preview 1](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) for any designer work you want to do.

Reviewer: ```suggestion
- [Visual Studio 2019 16.5 Preview 1](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the latest [Preview version of Visual Studio](https://visualstudio.microsoft.com/vs/preview/)
```

> - [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) 3.0 or later.

Reviewer 2: a few more comments:
Given that now we're adding the requirement to use Preview 1 here, then pre-reqs should state the version and link to the preview download. Also, since VS and workloads are a pre-req, you shouldn't need to install the SDK separately.

> [!NOTE]

> > **Visual Studio 2017** doesn't support .NET Core 3.0 projects. **Visual Studio 2019** supports .NET Core 3.0 projects. Starting with **Visual Studio 2019 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.

Reviewer 3: ```suggestion
> **Visual Studio 2017** doesn't support .NET Core 3.0 projects. **Visual Studio 2019** supports .NET Core 3.0 projects. Starting with **Visual Studio 2019 version 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.
```

> > **Visual Studio 2017** doesn't support .NET Core 3.0 projects. **Visual Studio 2019** supports .NET Core 3.0 projects. Starting with **Visual Studio 2019 version 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.

Reviewer 2: ```suggestion
> .NET Core 3.0 projects are only supported in **Visual Studio 2019** or a later version. Starting with **Visual Studio 2019 version 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.
```

>

> > To enable the designer, Go to **Tools** > **Options** > **Environment** > **Preview Features** and select the **Use the preview Windows Forms designer for .NET Core apps** option.

Reviewer 2: ```suggestion
> To enable the designer, go to **Tools** > **Options** > **Environment** > **Preview Features** and select the **Use the preview Windows Forms designer for .NET Core apps** option.
```

### Consider

> 

Reviewer 2: probably line 58 should also be changed. and ms.date updated since VS 2019 16.5 was launched after March 2019.
